gui/wxpython/datacatalog/tree.py

```diff
@@ -854,22 +862,112 @@ def OnRenameLocation(self, event):
     def OnStartEditLabel(self, node, event):
```

Comment:
In this function, remove all the notifications. Here we simply won't inform user they can't edit. We only inform user once they start editing and something goes wrong (in OnEditLabel).